src/frontends/pytorch/src/op_table.cpp

```diff
@@ -483,6 +483,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_ts() {
         // aten::imag - Supported in limited set of patterns
         // aten::index - Supported in limited set of patterns
         {"aten::index_copy_", op::inplace_op<op::translate_index_copy_>},
+        {"aten::index_fill_", op::inplace_op<op::translate_index_fill_>},
```

Comment:
Also need to declare it with `OP_CONVERTER` macros

Reply:
ok sure

Reply:
sorry as its my first pr im ill informed

Reply:
hi pls review